### PR TITLE
added camlimages.5.0.0

### PR DESCRIPTION
**Pull request by camlspotter**

### CODE REVIEW

**Comment by camlspotter** [comment]
DNS failure?  I cannot check the logs:

```
This site can’t be reached
https://ci.ocamllabs.io/ocaml/opam-repository/commit/7b9b484fe30fd2c8c4b826f938e6837c5e72a4a0?test=3%20V2.0%20Compilers is unreachable.
ERR_ADDRESS_UNREACHABLE
```

File: packages/camlimages/camlimages.5.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+maintainer: "jun.furuse@gmail.com"
+authors: ["Jun Furuse" "François Pessaux" "Pierre Weis"]
+homepage: "https://bitbucket.org/camlspotter/camlimages"
+bug-reports:
+  "https://bitbucket.org/camlspotter/camlimages/issues?status=new&status=open"
+license: "LGPL-2 with OCaml linking exception"
+dev-repo: "hg://https://bitbucket.org/camlspotter/camlimages"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "ocamlfind" {build}
```
Comment (rgrinberg) [inline packages/camlimages/camlimages.5.0.0/opam]: Customary remind that `ocamlfind` is likely not necessary :)

**Comment by camlspotter** [comment]
camlimages requires ocamlfind to check graphics library availability.

**Comment by rgrinberg** [comment]
camlspotter <notifications@github.com> writes:

> camlimages requires ocamlfind to check graphics library availability.

I see. From what I can tell, you're only using ocamlfind to test for the
availability of a library. You can actually ask dune to do that for you:

Just run your config program with something like:

(run ./ciconfig.exe -graphics ${lib-available:graphics})

I know this isn't as general as the extension to the configurator
library that you've made. But perhaps it's worth it to avoid ocamlfind.

By the way, we're merging configurator into dune (for 1.0) and are
planning on making some improvements. If you have any suggestions to
make, or improvements to suggest, don't hesitate to make them here:

https://github.com/ocaml/dune/pull/467

**Comment by camlspotter** [comment]
I am afraid dune cannot handle the problem of `graphics` for now, since dune thinks graphics library always exists, since its `META` file is installed even if `graphics.cm*` are not.  I think it is a bug of findlib.

My small effort here is to workaround this problem by actually testing a link of graphics to control the build of the optional library.  I am not successful at this point.  

I have created a ticket https://github.com/ocaml/dune/issues/563 for my problem.  I appreciate a lot if you could have some time to check it.  Thanks!

**Comment by camlspotter** [comment]
@rgrinberg ,  I will propose things at the ticket, once after I fix the issues of camlimages.

**Comment by rgrinberg** [comment]
Thanks camlspotter, this package looks good (the ci issues are customary). Congratulations on yet another port to dune.

**Comment by camlspotter** [comment]
Not obliged to fix the rev-dep failures?

**Pull request merged**